src/core/format.js

```diff
@@ -1,6 +1,8 @@
 import { tf } from '../locale/locale';
 
-const formatStringRender = v => v;
+require('any-date-parser');
```

Comment:
Shouldn't this be import instead of require?

Reply:
good call. fixed it